zedd-app/src/AppState.ts

```diff
@@ -98,7 +98,7 @@ export class Task {
 
   @serializable
   @observable
-  public clarityTaskIntId: number | undefined
+  public platformTaskIntId: number | undefined
```

Comment:
Diese Information speichert, welcher Task verwendet wird (intId). Diese müsste entweder eindeutig sein, indem man ein Prefix wie REPLICON_ oder OTT_ hinzufügt oder man ergänzt hier um ein neues den platformTyp, damit man über die Tasks den richtigen Task finden kann. Ich würde hier eher das neue Feld hinzufügen. Was meinst du? Dann müsste man alle Stellen, diesen platformTaskIntId verwenden zum Auflösen des Tasks anpassen, dass zusätzlich der platformTyp auch beachtet wird.
Lass uns bevor du das umsetzt mal sprechen.

Reply:
Wird in separatem PR bearbeitet: https://github.com/tobka777/zedd2/pull/125